src/jni/env.cpp

```diff
@@ -38,8 +38,36 @@ namespace jni {
         return env->ExceptionCheck();
     }
 
-    void Env::exception_describe() {
+    String Env::exception_describe() {
+#ifdef TOOLS_ENABLED
```

Comment:
Why is that Tool only and not debug ?

Reply:
Because I'm a dum dum and somehow had in mind that games run from the editor are tools_enabled. Will fix this